Source/CesiumRuntime/Private/Tests/CesiumCameraManager.spec.cpp

```diff
@@ -0,0 +1,76 @@
+#include "CesiumCameraManager.h"
+#include "Engine/Engine.h"
+#include "Engine/World.h"
+#include "Misc/AutomationTest.h"
+
+BEGIN_DEFINE_SPEC(
+    CesiumCameraManagerSpec,
```

Comment:
Just a drive-by comment, since I was looking at this to write my own unit tests.

In Unreal, structs are often prefixed with F, like `FStructName`. I looked at an example spec `AutomationDriver.spec.cpp` and it also defines the spec struct with an F prefix. So let's add it here for consistency.

```suggestion
    FCesiumCameraManagerSpec,
```

Also, can the naming scheme be updated for `TestGeoTransforms.cpp` as well?

Reply:
Committed changes to address both points. Thanks!